# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/builtin/TestingFunctions.h
# Commit: 3c0037ba793b
# Full Hash: 3c0037ba793b7d393449036987767a44423bf346
# Author: Jason Orendorff <jorendorff@mozilla.com>
# Date: 2018-02-24 10:13:47
# Regressor Bug: 1440431
# File Overlap Count: 1
# Description:
#   Bug 1440431 - Part 1: Move ValueToScript into the engine. r=nbp.
# ==============================================================================

diff -r 37f8ff509e9d -r 3c0037ba793b js/src/builtin/TestingFunctions.h
--- a/js/src/builtin/TestingFunctions.h	Fri Feb 23 09:01:08 2018 +0700
+++ b/js/src/builtin/TestingFunctions.h	Thu Feb 22 12:31:01 2018 -0600
@@ -20,6 +20,9 @@
 MOZ_MUST_USE bool
 testingFunc_assertRecoveredOnBailout(JSContext* cx, unsigned argc, Value* vp);
 
+extern JSScript*
+TestingFunctionArgumentToScript(JSContext* cx, HandleValue v, JSFunction** funp = nullptr);
+
 } /* namespace js */
 
 #endif /* builtin_TestingFunctions_h */